nuget/helpers/lib/NuGetUpdater/NuGetUpdater.Core.Test/Update/UpdateWorkerTestBase.cs

```diff
@@ -8,7 +8,7 @@
 
 namespace NuGetUpdater.Core.Test.Update;
 
-public class UpdateWorkerTestBase
+public abstract class UpdateWorkerTestBase
```

Comment:
Not really related to the rest of the PR but I noticed this and wanted to make sure we were explicit that you should only be extending this class.